concise-mid-tag.cddl

```diff
@@ -35,6 +34,34 @@ module-metadata-map = {
   * $$module-metadata-map-extension
 }
 
+module-entity-map = {
+  comid.entity-name => text,
+  ? comid.reg-id => any-uri,
+  comid.role => one-or-more<$module-role>,
+  * $$module-entity-map-extension
+}
+
+$module-role /= module-tag-creator
+$module-role /= module-creator
+$module-role /= module-distributor
+$module-role /= module-licensor
+$module-role /= module-maintainer
```

Comment:
```suggestion
$module-role /= comid.tag-creator
$module-role /= comid.creator
$module-role /= comid.distributor
$module-role /= comid.licensor
$module-role /= comid.maintainer
```

Reply:
This expands nicely considering the base name of module-role 
so each module-role could be either module-tag-creator
module-creator etc. I felt it a bit more clearer! But happy to consider your suggestion if others in the review agree!

Reply:
I think we decided to always use the `comid.` prefix in cases like the above.  Instead of the `module-` infix, I'd rather use `role`, so for example: `comid.role-distributor`

Reply:
ok, then better to stick with your original comment!